### ISSUE TITLE
Running example throws exception

### ISSUE THREAD

**Issue by philvarner** [stactools-packages/naip#26]
Running the example in the README

```
aws s3 cp --request-payer requester s3://naip-analytic/va/2018/60cm/fgdc/37077/m_3707763_se_18_060_20180825.txt .
export AWS_REQUEST_PAYER='requester'
stac naip create-item --fgdc m_3707763_se_18_060_20180825.txt \
  VA 2018 s3://naip-analytic/va/2018/60cm/rgbir_cog/37077/m_3707763_se_18_060_20180825.tif \
  json-dest/
```  

Throws the exception:

```
File "/Users/philvarner/.pyenv/versions/3.9.14/lib/python3.9/site-packages/stactools/naip/stac.py", line 105, in create_item
    fgdc_metadata_text = pystac.STAC_IO.read_text(fgdc_metadata_href)
AttributeError: module 'pystac' has no attribute 'STAC_IO'
```

**Comment by philvarner** [comment]
Issue seems to be that the stactools-naip package was last published 1.5 years ago to pypi?

**Comment by geomatician** [comment]
This issue can be closed as the package was updated to v 0.2.0 through PR [https://github.com/stactools-packages/naip/pull/31](url)

**Comment by gadomski** [comment]
Confirmed, example works for me. Closing.
